osgp/shared/osgp-ws-smartmetering/src/main/resources/SmartMeteringConfiguration.wsdl

```diff
@@ -709,698 +733,747 @@
       <wsdl:input message="tns:SetClockConfigurationAsyncRequest"
-                  name="SetClockConfigurationAsyncRequest">
+        name="SetClockConfigurationAsyncRequest">
       </wsdl:input>
       <wsdl:output message="tns:SetClockConfigurationResponse"
-                   name="SetClockConfigurationResponse">
+        name="SetClockConfigurationResponse">
       </wsdl:output>
     </wsdl:operation>
 
     <wsdl:operation name="ConfigureDefinableLoadProfile">
       <wsdl:input message="tns:ConfigureDefinableLoadProfileRequest"
-                  name="ConfigureDefinableLoadProfileRequest">
+        name="ConfigureDefinableLoadProfileRequest">
       </wsdl:input>
       <wsdl:output message="tns:ConfigureDefinableLoadProfileAsyncResponse"
-                   name="ConfigureDefinableLoadProfileAsyncResponse">
+        name="ConfigureDefinableLoadProfileAsyncResponse">
       </wsdl:output>
     </wsdl:operation>
     <wsdl:operation name="GetConfigureDefinableLoadProfileResponse">
       <wsdl:input message="tns:ConfigureDefinableLoadProfileAsyncRequest"
-                  name="ConfigureDefinableLoadProfileAsyncRequest">
+        name="ConfigureDefinableLoadProfileAsyncRequest">
       </wsdl:input>
       <wsdl:output message="tns:ConfigureDefinableLoadProfileResponse"
-                   name="ConfigureDefinableLoadProfileResponse">
+        name="ConfigureDefinableLoadProfileResponse">
       </wsdl:output>
     </wsdl:operation>
 
     <wsdl:operation name="SetMbusUserKeyByChannel">
       <wsdl:input message="tns:SetMbusUserKeyByChannelRequest"
-                  name="SetMbusUserKeyByChannelRequest">
+        name="SetMbusUserKeyByChannelRequest">
       </wsdl:input>
       <wsdl:output message="tns:SetMbusUserKeyByChannelAsyncResponse"
-                   name="SetMbusUserKeyByChannelAsyncResponse">
+        name="SetMbusUserKeyByChannelAsyncResponse">
       </wsdl:output>
     </wsdl:operation>
 
     <wsdl:operation name="GetSetMbusUserKeyByChannelResponse">
       <wsdl:input message="tns:SetMbusUserKeyByChannelAsyncRequest"
-                  name="SetMbusUserKeyByChannelAsyncRequest">
+        name="SetMbusUserKeyByChannelAsyncRequest">
       </wsdl:input>
       <wsdl:output message="tns:SetMbusUserKeyByChannelResponse"
-                   name="SetMbusUserKeyByChannelResponse">
+        name="SetMbusUserKeyByChannelResponse">
+      </wsdl:output>
+    </wsdl:operation>
+
+    <wsdl:operation name="GetKeys">
+      <wsdl:input message="tns:GetKeysRequest"
+        name="GetKeysRequest">
+      </wsdl:input>
+      <wsdl:output message="tns:GetKeysAsyncResponse"
+        name="GetKeysAsyncResponse">
+      </wsdl:output>
+    </wsdl:operation>
+
+    <wsdl:operation name="GetGetKeysResponse">
+      <wsdl:input message="tns:GetKeysAsyncRequest"
+        name="GetKeysAsyncRequest">
+      </wsdl:input>
+      <wsdl:output message="tns:GetKeysResponse"
+        name="GetKeysResponse">
```

Comment:
This file looks like it has hundreds of lines with noise distracting from the actual changes.
Now if someone else edits this file, all those pesky changes could go back the other way forming an annoying distraction once again.
As long as we don't have tightened the formatting conventions across the IDEs I think it would be a much better way to cooperate if we don't mix actual changes with such large scale reformatting ping pong, and try to stick with the conventions that were applied before.

Reply:
I agree we should not ping pong. On the other hand, the other wsdls were already formatted this way. So changing back this file would mean that it would be the odd one out. I would prefer to keep all wsdls formatted the same.

Reply:
Why do you say you agree, if you clearly don't? There is no ping pong without a ping, and you obviously don't see an issue with pinging here, and once you ping, you have started playing the game.

Reply:
Reverted the formatting